cl/simple_pages/templates/v2_help/coverage_fds.html

```diff
@@ -0,0 +1,118 @@
+    <p>This was a monumental change in policy, and since that time, <strong>Free Law Project has requested all of the judicial disclosures that are legally available</strong>. The Ethics in Government Act also requires certain judicial personnel to complete financial disclosures. We have requested these as well, but asked that the request be fulfilled as a lower priority than the judicial disclosures.
+    </p>
+    <p>At this time, we have received over <a class="underline" href="https://x.com/FreeLawProject/status/1301921783151341568">a dozen thumb drives from the Financial Disclosures Office</a>.
+    </p>
+    <img src="{% static "jpeg/disclosure-thumb-drives.jpeg" %}"
+         width="992"
+         height="309"
+         class="img-responsive img-rounded"
+         alt="Picture of thumb drives"
+    >
+    <p>Additionally, judicial nominees file financial disclosures with the Senate Judiciary Committee during the nomination process. Free Law Project <a class="underline" href="https://www.github.com/freelawproject/nomination-extractor/">built a machine learning model and tool</a> that could identify financial disclosures and used it to scan hundreds of thousands of pages of Senate records.
+    </p>
+    <p><a class="underline" href="https://www.judicialwatch.org/" rel="nofollow">One other organization</a> has systematically placed disclosures online, and we have gathered their corpus as well, as described below. A news organization called ABPNews also gathered them in the late nineties, but after going out of business in 2000, <a class="underline" href="https://x.com/FreeLawProject/status/1093612415185350656">their records were lost to water damage</a>. If you know of other sources, <a class="text-primary-600" href="{% url "contact" %}">please get in touch</a>.
+    </p>
+    <p>In 2023, ProPublica utilized our dataset for an ambitious investigation into the Supreme Court, which led to their winning the prestigious public service Pulitzer Prize in 2024. The Pulitzer committee <a class="underline" href="https://www.propublica.org/article/pulitzer-prize-announcement-propublica-supreme-court">praised</a> ProPublica's "groundbreaking and ambitious reporting that pierced the thick wall of secrecy surrounding the Supreme Court." This recognition underscores the critical importance of judicial transparency and the impact of our database in supporting investigative journalism.
+    </p>
+  </c-layout-with-navigation.section>
+
+  <c-layout-with-navigation.section id="totals">
+    <h2>Yearly Totals</h2>
+    <c-data-table
+      :columns="[
+        {'label': 'Source', 'field': 'source'},
+        {'label': 'Year', 'field': 'year'},
+        {'label': 'File Count*', 'field': 'count'},
+        {'label': 'File Size (GB)*', 'field': 'size'},
+        {'label': 'Notes', 'field': 'notes'},
+      ]"
+      :rows="[
+        {'source': 'Senate Records', 'year': '1987-2013', 'count': '874', 'size': '394.7 MB', 'notes': 'Extracted from Senate Records'},
```

Comment:
v1 version of this help page included a link in the notes of this row, pointing to: https://free.law/2021/12/22/nomination-disclosures. I've noticed this link is no longer used in v2. is this intentional?